api/src/handlers/remove.handler.ts

```diff
@@ -11,6 +14,8 @@ const remove = async ctx => {
     return;
   }
 
+  cache.set(`pokemon-${id}`, null)
```

Comment:
Shouldn't we remove the entry instead of setting it to `null`?

Reply:
It makes sense, I'll add these commands to our cache.